src/server.js

```diff
@@ -1,31 +1,32 @@
+import filter from '@arr/filter'
```

Comment:
Let's drop the filter. I don't want anyone to accidentally use it when mutations matter.